pkg/kopia/cli/repository/storage/s3/arg_test.go

```diff
@@ -0,0 +1,68 @@
+package s3
+
```

Comment:
Is `ArgTest` a utilitity to be used across different packages. This doesnt seem relevant to s3 package. Should we move it to a different package or utils package?

Reply:
Agreed. I will move it to [pkg/kopia/cli/internal/test](https://github.com/kanisterio/kanister/blob/be4031d37595569244239a5068af77470b83ca97/pkg/kopia/cli/internal/test/)